docs/source/changelog.rst

```diff
@@ -5,6 +5,13 @@ Changelog
 
 All notable changes to this project will be documented in this file.
 
+`0.28.2`_ -- 2020-04-20
+-----------------------
+Fixed
+^^^^^
+* Typing: fix typing for actor overload and add types for broker, result and scheduler
+* Test: test_actors_can_be_canceled was not deterministic
```

Comment:
Generally I do not add this into the changelog, To my mind, Changelog is for developers using the library, they don't care about the test. What do you think ?